static/sass/styles.scss

```diff
@@ -4,10 +4,9 @@ $color-brand: #e95420;
 $color-accent: #2c001e;
 
 // global components
-@import 'modules/main';
 @import 'modules/cookie-policy';
 
-// import vanilla-brochure-theme
+// import vanilla
```

Comment:
So much attention to detail :open_mouth:

Reply:
Look after the pennies and the pounds look after themselves. ;)